src/components/TakeQuizPage/index.jsx

```diff
@@ -190,58 +190,77 @@ const RadioOptions = () => {
     const setSelectedOption = (event) => {
         updateSelectedOptions(questionId, [event.target.value]);
     };
+    const clearSelection = (event) => {
+        updateSelectedOptions(questionId, [event.target.value]);
```

Comment:
You can destructure `clearSelectedOptions` from the handler context and use that instead of `updateSelectedOptions`. It only takes the questionId.